libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -944,7 +944,6 @@ ap_message GCS_MAVLINK::mavlink_id_to_ap_message_id(const uint32_t mavlink_id) c
         { MAVLINK_MSG_ID_BATTERY2,              MSG_BATTERY2},
         { MAVLINK_MSG_ID_CAMERA_FEEDBACK,       MSG_CAMERA_FEEDBACK},
 #if HAL_MOUNT_ENABLED
-        { MAVLINK_MSG_ID_MOUNT_STATUS,          MSG_MOUNT_STATUS},
```

Comment:
Where is the replacement entry here for the replacement message?

Reply:
txs, added the mapping between MAVLINK_MSG_ID_DEVICE_ATTITUDE_STATUS and MSG_DEVICE_ATTITUDE_STATUS.